dao-utils/src/test_env.rs

```diff
@@ -97,6 +97,10 @@ impl TestEnv {
     pub fn advance_block_time_by(&self, seconds: Duration) {
         self.state.lock().unwrap().block_time += seconds.as_secs();
     }
+
+    pub fn get_current_block_time(&self) -> u64 {
```

Comment:
I think, we can omit `current`, `block_time` is current by default
```suggestion
    pub fn get_block_time(&self) -> u64 {
```